README.md

```diff
@@ -1,8 +1,107 @@
+Table of Contents
+=================
 
+* [Introduction](#introduction)
+* [Charts repository](#repository)
+  * [Available Charts in Repository](#available-charts-in-repository)
+  * [Usage](#usage)
+  * [Development](#development)
+  * [Testing](#testing)
+* [Templates](#templates)
+  * [Role-Based Access Control](#rbac)
 
-## How to play
+# Introduction
+
+This repository contains Helm templates and charts used by other applications part of the Snowdrop team.
+
+# Repository
+
+The repository has been configured to serve the static helm index and chart files. To use it:
+
+```console
+$ helm repo add snowdrop http://snowdrop.github.io/helm
+```
+
+And confirm that the snowdrop repository is listed:
+
+```console
+$ helm repo list
+NAME           	URL                               
+snowdrop	    http://snowdrop.github.io/helm
+```
+
+## Available Charts in Repository
+
+| Chart Name                     | Description | Configuration |
+|--------------------------------|-------------| --------------|
+| spring-boot-example-app        | Chart to be used by the Snowdrop Spring Boot examples | [values.yaml](repository/spring-boot-example-app/values.yaml) |
+
+## Usage
+
+To use one of the available charts, for example `spring-boot-example-template`, update or add the `requirements.yaml` file to register the dependency:
```

Comment:
Where is such a `requirements.yaml` file located ?

Reply:
Changed. Let me know if it looks clearer now.